### PR TITLE
New package: arma3-unix-launcher-git-20200112_1

### PR DESCRIPTION
**Pull request by Auberginez**

### CODE REVIEW

**Comment by Noah-Huppert** [comment]
I think the build failed due to some reason unrelated to this package:

```
=> ERROR: xbps-src: failed to update xbps!
````

I built and installed this package just fine on x86_64.

File: srcpkgs/arma3-unix-launcher-git/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'arma3-unix-launcher-git'
+pkgname=arma3-unix-launcher-git
+version=20200112
+revision=1
+_commit=facedb0993d66e7bf31fde87e945a59641d25e43
+wrksrc="arma3-unix-launcher-$_commit"
+build_style=cmake
+makedepends="cmake pkg-config gtkmm-devel"
```
Comment (abenson) [inline srcpkgs/arma3-unix-launcher-git/template:8]: drop cmake here, move pkg-config to hostmakedepends

**Comment by ahesford** [comment]
Void does not accept `-git` packages. It looks like upstream tagged a release nine hours ago, so you can update to fetch the release. However, the upstream numbering scheme is based on commit number and truncated hash. This could be fragile; I recommend you file an issue upstream to provide more sensible numbering. (If the maintainer ever decides to adopt more conventional release numbering, it will likely break XBPS version comparisons and require some ugly workarounds.)

Also, the other archs failed CI for legitimate issues, not just XBPS failures. These will need to be resolved before the package can be considered for inclusion.

File: srcpkgs/arma3-unix-launcher-git/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'arma3-unix-launcher-git'
+pkgname=arma3-unix-launcher-git
+version=20200112
```
Comment (abenson) [inline srcpkgs/arma3-unix-launcher-git/template:3]: just use the versions [they are](https://github.com/muttleyxd/arma3-unix-launcher/releases). version=235

File: srcpkgs/arma3-unix-launcher-git/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+_commit=facedb0993d66e7bf31fde87e945a59641d25e43
+wrksrc="arma3-unix-launcher-$_commit"
```
Comment (abenson) [inline srcpkgs/arma3-unix-launcher-git/template:6]: wrksrc="arma3-unix-launcher-commit-${version}"

File: srcpkgs/arma3-unix-launcher-git/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'arma3-unix-launcher-git'
+pkgname=arma3-unix-launcher-git
+version=20200112
+revision=1
+_commit=facedb0993d66e7bf31fde87e945a59641d25e43
```
Comment (abenson) [inline srcpkgs/arma3-unix-launcher-git/template:5]: Don't need this.

File: srcpkgs/arma3-unix-launcher-git/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+wrksrc="arma3-unix-launcher-$_commit"
+build_style=cmake
+makedepends="cmake pkg-config gtkmm-devel"
+depends="gtkmm"
+short_desc="ArmA 3 Launcher for Linux and Mac"
+maintainer="Auberginez <aubergines@protonmail.com>"
+license="MIT"
+homepage="https://github.com/muttleyxd/arma3-unix-launcher"
+distfiles="${homepage}/archive/${_commit}.tar.gz"
```
Comment (abenson) [inline srcpkgs/arma3-unix-launcher-git/template:14]: distfiles="${homepage}/archive/commit-${version}.tar.gz"

File: srcpkgs/arma3-unix-launcher-git/template [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'arma3-unix-launcher-git'
+pkgname=arma3-unix-launcher-git
```
Comment (abenson) [inline srcpkgs/arma3-unix-launcher-git/template:2]: drop the -git

**Comment by ericonr** [comment]
Author hasn't been active for a while, closing for now. Feel free to reopen!

**Pull request closed without merging**